### ISSUE TITLE
Question about ppo algorithm

### ISSUE THREAD

**Issue by sinaqahremani** [Acmece/rl-collision-avoidance#21]
Hello,
I have 2 questions about the implementation of PPO.

what is `dist_entropy` used in **evaluate_action** method of CNNPolicy implemented in net.py and in **ppo_update_stage1** function?

And I need theory background of this line in ppo.py:
`loss = policy_loss + 20 * value_loss - coeff_entropy * dist_entropy`
